Sources/Rendering/Core/ScalarBarActor/index.js

```diff
@@ -355,17 +340,11 @@ function vtkScalarBarActorHelper(publicAPI, model) {
       model.tmContext.fillText(key, 1, value.startingHeight + value.height - 1);
     });
 
-    const image = new Image();
-    image.src = model.tmCanvas.toDataURL('image/png');
-    model.nextImage = image;
-    model._nextAtlas = newTmAtlas;
-    if (image.complete) {
-      publicAPI.completedImage(false);
-    } else {
-      image.addEventListener('load', () => {
-        publicAPI.completedImage(true);
-      });
-    }
+    model.tmContext.getImageData(0, 0, maxWidth, totalHeight);
```

Comment:
Yeh that was there to test something related to ``` // In Chrome 69 on Windows and Ubuntu, there is a bug that prevents some
    // canvases from working properly with webGL textures.  By getting any
    // image data from the canvas, this works around the bug.  See
    // https://bugs.chromium.org/p/chromium/issues/detail?id=896307
    if (navigator.userAgent.indexOf('Chrome/69') >= 0) {
      ctx.getImageData(0, 0, 1, 1);
    }```
    which we still have. I'll remove both as that is an old chrome at this point

Reply:
In that case, I'm fine keeping this here as-is. If you want to keep it, I would suggest adding a comment explaining its purpose, since otherwise it looks quite unnecessary.